packages/contracts/deploy/000-L2OutputOracle.deploy.ts

```diff
@@ -0,0 +1,26 @@
+/* Imports: Internal */
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+import 'hardhat-deploy'
+
+const deployFn: DeployFunction = async (hre) => {
+  const { deploy } = hre.deployments
+  const { deployer } = await hre.getNamedAccounts()
+
+  await deploy('L2OutputOracle', {
+    from: deployer,
+    args: [
+      15,
+      2,
+      `0x${'00'.repeat(32)}`,
+      0,
+      process.env.L2OO_STARTING_BLOCK_TIMESTAMP,
```

Comment:
This could be hardened by adding a fallback value if its not set, maybe checking to see the timestamp on the latest block?

Reply:
IMO the deployment should fail if that value isn't set rather than falling back. I can update that.